docs/getting_started.md

```diff
@@ -31,14 +34,14 @@ Along with AWS UPI prerequisites we need following
 The system `sed` package is not compatible with the script used to install AWS
 UPI. To resolve this issue, you must install `gnu-sed`. You can do this with brew.
 
-    brew install gnu-sed
+    `brew install gnu-sed`
```

Comment:
when I view the file, I see backtick for this command but not for other code blocks, not sure what is missing.

Reply:
@vasukulkarni : I restored this back to the way it was.